android/library/src/main/java/com/liferay/mobile/screens/auth/login/interactor/LoginCookieInteractor.java

```diff
@@ -22,9 +23,17 @@ public BasicEvent execute(Object... args) throws Exception {
 		String password = (String) args[1];
 		Authenticator authenticator = (Authenticator) args[2];
 
+		boolean shouldHandleExpiration = (boolean) args[3];
```

Comment:
Does it makes sense to make them required in every call?

Reply:
This is the only way to configure this attribute, what do you suggest?